CONTRIBUTING.md

```diff
@@ -0,0 +1,37 @@
+# How to Contribute
+
+Contributing to this project entails submitting pull requests (PRs) to this repository. PRs then need to be appropriately assigned and labelled by someone with the appropriate permissions.
```

Comment:
In general, we should rely on a more detailed contributing guide similar to https://github.com/apache/spark/blob/master/CONTRIBUTING.md -- but I don't think we need all of that yet since that's a much larger project.

Reply:
Added instruction to mention a general category in the PR. The categories are:

- Bug Fix
- Feature
- Documentation
- New/updated tests